app/component/itinerary/ItineraryDetails.js

```diff
@@ -262,6 +264,24 @@ class ItineraryDetails extends React.Component {
       }
     }
 
+    if (config.showRouteDisclaimer) {
+      itinerary.legs.forEach(leg => {
+        const { route } = leg;
+        if (
+          route?.desc?.length > 0 &&
```

Comment:
This sometimes evaluates as `undefined > 0`, which feels uncomfortable. How about dropping the comparison?
`route?.desc?.length` would skip undefined and value 0.